### PR TITLE
Deprecate passing individual tables to `amici_import_petab`

### LINKED ISSUE #2460: Replace deprecated `petab.Problem.from_files` in `amici.petab.cli.import_petab._main`
Replace deprecated `petab.Problem.from_files` in `amici.petab.cli.import_petab._main`.

Also deprecate passing individual PEtab tables to `amici_import_petab` on the command line (who is doing that anyways?), but while it's still in place, check that the parameter table is present.

### PR DESCRIPTION
**Pull request by dweindl**
and replace deprecated `petab.Problem.from_files` in `amici.petab.cli.import_petab._main`.

Fixes #2460

### CODE REVIEW

File: python/sdist/amici/petab/cli/import_petab.py [hunk 1]
```diff
@@ -102,18 +115,42 @@ def _parse_cli_args():
     )
 
     args = parser.parse_args()
```
Comment (FFroehlich) [inline python/sdist/amici/petab/cli/import_petab.py:117]: don't we need a a `group.parse_args()` as well?

File: python/sdist/amici/petab/cli/import_petab.py [hunk 2]
```diff
@@ -102,18 +115,42 @@ def _parse_cli_args():
...
-
-    if not args.yaml_file_name and not all(
+    if any(
+        [
+            args.sbml_file_name,
```
Comment (FFroehlich) [inline python/sdist/amici/petab/cli/import_petab.py:120]: parser doesn't have them as options?

**Review by FFroehlich — APPROVED** [review]
Maybe add tests for both calls?

File: python/sdist/amici/petab/cli/import_petab.py (hunk 1 above)
Comment (dweindl) [reply python/sdist/amici/petab/cli/import_petab.py:117]: No, that's happening automatically

File: python/sdist/amici/petab/cli/import_petab.py (hunk 2 above)
Comment (dweindl) [reply python/sdist/amici/petab/cli/import_petab.py:120]: It does, all arguments from `parser.add_argument_group().add_argument()` will be available through that  `parser`.

Comment (FFroehlich) [reply python/sdist/amici/petab/cli/import_petab.py:120]: ah :)

**Comment by dweindl** [comment]
> Maybe add tests for both calls?

the yaml-based one is used for the benchmark collection, added one for the deprecated call

**Pull request merged**